keras/backend/tensorflow_backend.py

```diff
@@ -11,7 +11,7 @@
 _SESSION = None
 _LEARNING_PHASE = tf.placeholder(dtype='uint8', name='keras_learning_phase')  # 0 = test, 1 = train
 _MANUAL_VAR_INIT = False
-
+_DEVICE = None
```

Comment:
Do we really have to rely on a global variable to implement device scopes? What would be some other cleaner options?